src/js/lite/ui/TooltipUI.js

```diff
@@ -40,13 +40,15 @@ class TooltipUI {
   show() {
     const $node = this.$node;
     const offset = $node.offset();
+    const targetOffset = $(this.options.target).offset();
```

Comment:
`this.options.target` is only one property of multi summernote instances. 

So tooltip will be show in first summernote instance.